packages/app/cypress/e2e/studio/studio.cy.ts

```diff
@@ -2,6 +2,12 @@ import { launchStudio, loadProjectAndRunSpec, assertClosingPanelWithoutChanges }
 
 describe('Cypress Studio', () => {
   function incrementCounter (initialCount: number) {
+    cy.waitForSpecToFinish(undefined, undefined, false)
+
+    cy.findByTestId('queued-icon').should('not.exist')
+    cy.get('.runnable-active').should('not.exist')
+    cy.contains('No commands were issued in this test.').should('not.exist')
```

Comment:
Seems odd to have this code in `incrementCounter`, is there a better location for it?